src/types/ApplicationFormField.ts

```diff
@@ -10,6 +10,7 @@ interface ApplicationFormField {
 	readonly baseField: BaseField;
 	position: number;
 	label: string;
```

Comment:
The DB allows null already, but this line needs updating to support null too:

`instructions: string | null`